### PR TITLE
Add display functionality

### PR DESCRIPTION
**Pull request by kgooris**
Hi this time including the missing files ;)

### CODE REVIEW

**Comment by mariolukas** [comment]
Sorry i will merge this soon. I am still testing with my ESP and an Oled Display.

**Comment by kgooris** [comment]
> 

Great! I am working on a version including bluetooth and mqtt icon on the screen also ;)

File: Bluetti_ESP32/display.cpp [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+
+
+
+#define SCREEN_WIDTH 128
+#define SCREEN_HEIGHT 64
+Adafruit_SSD1306 display(SCREEN_WIDTH, SCREEN_HEIGHT, &Wire, -1);
+
+void initDisplay()
+{
+
```
Comment (SmokyBob) [inline Bluetti_ESP32/display.cpp:41]: Code to manage oled reset if needed

```
#if OLED_RST
    pinMode(OLED_RST, OUTPUT);
    digitalWrite(OLED_RST, HIGH);
    delay(20);
    digitalWrite(OLED_RST, LOW);
    delay(20);
    digitalWrite(OLED_RST, HIGH);
    delay(20);
#endif
```

**Review by SmokyBob — COMMENTED** [review]
Pin Reset for some oled boards is needed.

P.S. first review in Github since a long time so I might change the comments if they are presented as code "ready to be integrated"

File: Bluetti_ESP32/config.h [hunk 2]
```diff
@@ -3,7 +3,13 @@
 #include "Arduino.h"
 
 #define DEBUG                 1
-//#define RESET_WIFI_SETTINGS   1
+// Display config section, comment DISPLAYSSD1306 to disable display
+//#define DEBUGDISP 1
+//#define DISPLAYSSD1306  1
+#define DISPLAY_SCL_PORT 4
+#define DISPLAY_SDA_PORT 5
+
```
[the remark this answers is not in the document]
Comment (kgooris) [reply Bluetti_ESP32/config.h:13]: added to my main branch

**Pull request merged**